cache.go

```diff
@@ -469,9 +469,9 @@ func (c *Cache[K, V]) Items() map[K]*Item[K, V] {
 
 	items := make(map[K]*Item[K, V], len(c.items.values))
 	for k := range c.items.values {
```

Comment:
I think we can get the item value here:
```suggestion
	for k, v := range c.items.values {
```

And then below (line 472), we'd just need to do a type cast/assertion:
```
item := v.Value.(*Item[K, V])
```

Reply:
Thanks for the feedback. Updated